### ISSUE TITLE
Trying to access array offset on value of type bool in Model/Item.php

### ISSUE THREAD

**Issue by JKingma** [magmodules/magento2-channable#109]
For one of our clients we run into the following error that prevents the feed to be generated correctly:
```
Channable.INFO: Generate: Notice: Trying to access array offset on value of type bool in 
.../vendor/magmodules/magento2-channable/Model/Item.php on line 118 [] []
```
After adding some logging it seems that in this case the $row is false. Adding an extra if statement fixes the problem temporary but it shouldn't be possible to get a false here.

We run version 1.4.20. 
The issue has been occured since we're on php 7.4 and mysql 8 with Magento 2.4.1-p1. 
I hope you can reproduce the issue.

**Comment by Marvin-Magmodules** [comment]
Hi @JKingma, this is fixed in 1.4.21, can you update to the latest version?

**Comment by JKingma** [comment]
Wow that's a quick reply. Thanks, we're going to upgrade!
